src/corpus.py

```diff
@@ -49,3 +50,9 @@ def load(cls, filename):
 
     def save(self, filename):
         pickle.dump(self, file(filename, 'wb'))
+
+# Mistake created:
+# - Line 29: 'list()' instead of '[]'
+#   - Slower and more obfuscated
+# - Line 48 & 51: 'file(...)' instead of 'open(...)'
+#   - Deprecated syntax
```

Comment:
Hi Parthiv! I really appreciate that you took the time to create mistake for me to find. Unfortunately, I think the answers were too easy to accidentally read, hehe.

For example, I caught the `file()` error, and went to make a comment, so I Ctrl+F'd for `file(`. But... I accidentally read this comment block in the process.

I think it might be better to just turn this back into a regular code review without the intentional mistakes. But, thank you again for taking the time!